src/index.html

```diff
@@ -2,11 +2,459 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width,
+      initial-scale=1.0"
+    >
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
+
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header-logo">
+        <img src="images/logo.png" class="logo"
+        alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__ul">
+          <li class="nav__item">
+            <a  class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a  class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" data-qa="nav-hover">Laptops & computers</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="main__card">
+        <article
+          class="card card--container"
+          data-qa="card"
+        >
+          <img
+            src="images/imac.jpeg"
+            class="card__image"
+            alt="APPLE-A1419-iMac-27-picture"
+          >
+
+          <h3 class="card__item-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+
+          <div class="card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__reviews">
+              Reviews: 5
+            </p>
+          </div>
```

Comment:
You can create div for card__stars and card__reviews  together. and in styles create for this div display: flex and flex-direction: row and align in the center. Your text and stats would be in the center.  You can use  semantic tags instead of div

Reply:
Heard ul li semantic better practice for this block.